src/middleware/middleware.id.ts

```diff
@@ -0,0 +1,6 @@
+import { createParamDecorator, ExecutionContext } from '@nestjs/common';
+
+export const IdFromJwt = createParamDecorator((_, context: ExecutionContext) => {
```

Comment:
I think this might be improve by sending the Auth headers instead on getting all the context. Let's meet up and check.